src/centos/stream9/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM quay.io/centos/centos:stream9-development
```

Comment:
This file is a copy of centos/8/Dockerfile, with 4 changes:

- Updated `FROM` line:
- `epel-release` isn't installed. It doesn't exist (yet).
- `powertools` -> `crb` rename. They are the same thing, just different repository names for different CentOS versions
-  `libidn-devel` -> `libidn2-devel`.